File: wexpt/app/src/main/java/net/wexpt/com/wexpt/ui/http/HttpRequestIn.java
package net.wexpt.com.wexpt.ui.http;

import android.content.Context;
import android.os.Handler;

import java.util.Map;

/**
 * Created by vvguoliang on 2017/9/22.
 * <p>
 * 返回数据接口
 */

public interface HttpRequestIn {

    void setPublic(Context context, Map<String, Object> map, Handler mHnadler, String url, String name);

    void setWIFI_ADDRESS(Context context, Handler mHandler);
}

File: wexpt/app/src/main/java/net/wexpt/com/wexpt/ui/http/HttpRequestIn.kt
package net.wexpt.com.wexpt.ui.http

import android.content.Context
import android.os.Handler

/**
 * @Time : 2018/4/17 no 下午4:21
 * @USER : vvguoliang
 * @File : HttpRequestIn.kt
 * @Software: Android Studio
 *code is far away from bugs with the god animal protecting
 *   I love animals. They taste delicious.
 * ***┏┓   ┏ ┓
 * **┏┛┻━━━┛ ┻┓
 * **┃   ☃   ┃
 * **┃ ┳┛  ┗┳ ┃
 * **┃    ┻   ┃
 * **┗━┓    ┏━┛
 * ****┃    ┗━━━┓
 * ****┃ 神兽保佑 ┣┓
 * ****┃ 永无BUG！┏┛
 * ****┗┓┓┏━┳┓┏┛┏┛
 * ******┃┫┫  ┃┫┫
 * ******┗┻┛  ┗┻┛
 */
interface HttpRequestIn {

    abstract fun setPublic(context: Context, map: Map<String, Any>, mHnadler: Handler, url: String, name: String)

    abstract fun setHOME(context: Context, mHnadler: Handler, result: String)
}